crates/sui-bridge/src/tools/cli.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
for some reasons it does not like to think this file was renamed from `validator_cli.rs`